app/controllers/article-controller.js

```diff
@@ -0,0 +1,20 @@
+const { fetchArticleById, fetchArticlesArr } = require("../models/article-model")
+
+exports.getArticleById = (req, res, next) => {
+    const id = req.params.article_id
```

Comment:
For slightly cleaner syntax you can destructure this like so: 
`const {article_id} = req.params`

You can then pass article_id to fetchArticleById